pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -86,6 +87,10 @@ public SegmentCreationJob(String jobName, Properties properties) throws Exceptio
       _port = Integer.parseInt(portString);
     }
     _tableName = _properties.getProperty(JobConfigConstants.SEGMENT_TABLE_NAME);
+    String enableHttps = _properties.getProperty(JobConfigConstants.ENABLE_HTTPS);
+    if (enableHttps != null) {
```

Comment:
Can you add `LOGGER.info` below where we print out all configs? (after `*********************`)